src/cli.js

```diff
@@ -226,7 +227,9 @@ function cli(inputArgs) {
                 downstreamArgs.push('--' + flag);
             }
         });
-
+        if (args.buildConfig) {
+            downstreamArgs.push('--buildConfig=' + args.buildConfig);
```

Comment:
ok, never mind. it seems like cordova-lib just passes the --buildConfig args to the underlying platform, which does the bulk of the work : https://github.com/apache/cordova-android/pull/164